tests/agent_features/test_ml_events.py

```diff
@@ -121,27 +125,31 @@ def _test():
     _test()
 
 
+@override_application_settings({"ml_insights_events.enabled": True})
 @reset_core_stats_engine()
 @validate_ml_event_count(count=0)
 @background_task()
 def test_record_ml_event_inside_transaction_bad_event_type():
     record_ml_event("!@#$%^&*()", {"foo": "bar"})
 
 
+@override_application_settings({"ml_insights_events.enabled": True})
 @reset_core_stats_engine()
 @validate_ml_event_count(count=0)
 def test_record_ml_event_outside_transaction_bad_event_type():
     app = application()
     record_ml_event("!@#$%^&*()", {"foo": "bar"}, application=app)
 
 
+@override_application_settings({"ml_insights_events.enabled": True})
 @reset_core_stats_engine()
 @validate_ml_event_count(count=0)
 @background_task()
 def test_record_ml_event_inside_transaction_params_not_a_dict():
     record_ml_event("ParamsListEvent", ["not", "a", "dict"])
 
 
+@override_application_settings({"ml_insights_events.enabled": True})
```

Comment:
I don't think all of the tests in this folder shouldn't need a decorator for each function. We can just add these settings to the defaults and default them all to on.